lib/ui/painting/image_descriptor.cc

```diff
@@ -15,6 +15,19 @@
 #include "third_party/tonic/dart_binding_macros.h"
 #include "third_party/tonic/logging/dart_invoke.h"
 
+#ifdef OS_MACOSX
```

Comment:
Nit: Please explicitly include `flutter/fml/build_config.h` as that adds these macros. Right now, we are depending on one of the headers including this header. But, if those are refactored, there is a chance we'll just silently flow through the else case.

Reply:
Done